lib/matplotlib/backends/backend_pdf.py

```diff
@@ -766,10 +766,30 @@ def __init__(self, filename, metadata=None):
         self.writeObject(self.resourceObject, resources)
 
     fontNames = _api.deprecated("3.11")(property(lambda self: self._fontNames))
-    dviFontInfo = _api.deprecated("3.11")(property(lambda self: self._dviFontInfo))
     type1Descriptors = _api.deprecated("3.11")(
         property(lambda self: self._type1Descriptors))
 
+    @_api.deprecated("3.11")
+    def dviFontInfo(self):
+        d = {}
+        tex_font_map = dviread.PsfontsMap(dviread.find_tex_file('pdftex.map'))
+        for pdfname, dvifont in self._dviFontInfo.items():
+            psfont = tex_font_map[dvifont.texname]
+            if psfont.filename is None:
+                raise ValueError(
+                    "No usable font file found for {} (TeX: {}); "
+                    "the font may lack a Type-1 version"
+                    .format(psfont.psname, dvifont.texname))
+            d[pdfname] = types.SimpleNamespace(
```

Comment:
Didn't the key use to be `dvifont.texname`?

Reply:
indeed, fixed.